huaweicloud/resource_huaweicloud_cce_addon_v3.go

```diff
@@ -53,10 +53,49 @@ func resourceCCEAddonV3() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"values": {
+				Type:     schema.TypeList,
+				Optional: true,
+				ForceNew: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"basic": {
+							Type:     schema.TypeMap,
+							Required: true,
+						},
+						"custom": {
+							Type:     schema.TypeMap,
+							Optional: true,
+						},
+					},
+				},
+			},
 		},
 	}
 }
 
+func getValuesValues(d *schema.ResourceData) (basic, custom map[string]interface{}, err error) {
+	values := d.Get("values").([]interface{})
+	if len(values) == 0 {
+		log.Printf("[WARNING] no values are set for CCE addon")
```

Comment:
Seems we don't need a WARNING here as values are commonly empty

Reply:
fixed